content/articles/2019-01-purrr-0-3-0.Rmd

```diff
@@ -0,0 +1,273 @@
+---
+title: 'purrr 0.3.0'
+author: Lionel Henry
+date: '2019-01-25'
+slug: purrr-0-3-0
+description: >
+  purrr 0.3.0 is now on CRAN.
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/NodtnCsLdTE
+  author: Mikhail Vasilyev
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.width = 7,
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+
+library("purrr")
+```
+
+We're excited to announce the release of purrr 0.3.0! This release polishes the consistency of the interface, introduces new `pluck()`-based functions, many improvements to mappers, and a set of tools to limit the rate at which a function is called. Find a detailed account of the changes in the [NEWS](https://github.com/tidyverse/purrr/blob/master/NEWS.md#purrr-030) file.
+
+
```

Comment:
I think we should always reveal all the necessary code, having watched some people live code based on quick Googling. So that means always making this explicit:

  * Install the latest version via `install.packages("purrr")`.
  * Attach (or load?) purrr via `library(purrr)`.

Reply:
Also needs a brief overview of the rest of the documentation. It currently feels like a grab bag of different features; that's fine, but we should prepare the user for it.

Reply:
Instead of attach/load: "Start working with purrr"?